### PR TITLE
update report content

### PR DESCRIPTION
**Pull request by LFISHER7**
This updates the report content:

* Reorganisaes the introduction (and renames this section)
* Adds in report limitations section
* Few other misc changes

### CODE REVIEW

File: interactive_templates/templates/v2/analysis/render_report.py [hunk 1]
```diff
@@ -173,13 +173,15 @@ def get_data(
     # population logic
 
     if population == "adults":
-        population_definition = "all registered patients aged 18 and over"
+        population_definition = "all patients aged 18 and over registered with a general practice at the start of each month"
 
     elif population == "children":
-        population_definition = "all registered patients aged under 18"
+        population_definition = "all patients aged under 18 registered with a general practice at the start of each month"
```
[the remark this answers is not in the document]
Comment (LFISHER7) [reply interactive_templates/templates/v2/analysis/render_report.py:179]: It's age at the start of the month

File: interactive_templates/templates/v2/analysis/report_template.html [hunk 2]
```diff
@@ -65,29 +137,33 @@ <h2>Table of contents</h2>
...
-                        measure of interest occurred (up to once per month, per
-                        patient), as well as the number of unique patients
-                        experiencing the event between {{ start_date }} and {{
-                        end_date }}. The number of events in the latest month is
-                        also shown.
+                    <p>The table below shows the total number of times the measure of interest
+                        occurred and the number of unique patients experiencing the event between
+                        {{ start_date }} and {{ end_date }}. The number of events in the latest
+                        complete month and latest week is also shown. An event is defined as a
+                        patient having a code recorded from both codelist 1 and codelist 2 that
```
[the remark this answers is not in the document]
Comment (LFISHER7) [reply interactive_templates/templates/v2/analysis/report_template.html:144]: Yes, good idea!

**Comment by LFISHER7** [comment]
Thanks for picking up those errors @lucyb !

File: interactive_templates/templates/v2/analysis/render_report.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (lucyb) [reply interactive_templates/templates/v2/analysis/render_report.py]: Hmm, maybe that sentence should therefore be:

> all patients who are aged under 18 and registered with a general practice at the start of each month

**Pull request merged**